servers/database/src/main/resources/db/migration/maps_db/V1.00.08__maps_repo_store_last_comit_date.sql

```diff
@@ -0,0 +1,12 @@
+/* recreate map_index table with 'last_commit_date' column rather than a 'version' column. */
```

Comment:
I'm actually surprised by this. I would've expected 2 alter table statements to modify the table, but if the table get's recreated every now and then this approach works too I guess :D

Reply:
Considering there is no data in the table, this was an okay route. I went this way so the column ordering made more sense, and was relatively easy all in all.